packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/interventions/InterventionValueType.java

```diff
@@ -0,0 +1,11 @@
+package software.uncharted.terarium.hmiserver.models.simulationservice.interventions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum InterventionValueType {
+	@JsonProperty("value")
```

Comment:
Nitpick:
I believe the convention of enums is to be all caps. That should just involve removing this `@JsonProperty`

Reply:
seconded!